clusters/build-clusters/01_cluster/hack/generate-bw-items.sh

```diff
@@ -14,31 +14,48 @@ readonly WORKDIR
 CLUSTER_NAME="build01"
 readonly CLUSTER_NAME
 
+SAVED_CURRENT_CONTEXT="$(oc config current-context)"
+readonly SAVED_CURRENT_CONTEXT
+
+function recover_current_context {
+  echo "recovering the current context"
+  kubectl config use-context "${SAVED_CURRENT_CONTEXT}"
+}
+
+trap recover_current_context EXIT
```

Comment:
Why do we need this?

Reply:
Removed.

I really do not want to change my current context by running this script.
So I wanted to recover it before exit.

I also tried with `--context` with all oc-calls in the script.
However, the most important one is not working with it.
> oc sa create-kubeconfig -n ci "${sa}" > "${config}"

Probably a bug of oc-cli ... but ...

Reply:
I think the recover is fine and Steve just wanted to understand it, not necesarily to change it

Reply:
I see. Thanks for the tips.

I will give it another try when it is fixed.
https://bugzilla.redhat.com/show_bug.cgi?id=1827463